ddtrace/contrib/trace_utils.py

```diff
@@ -165,6 +165,16 @@ def wrapper(wrapped, instance, args, kwargs):
     return with_mod
 
 
+def distributed_tracing_enabled(int_config, default=False):
+    # type: (IntegrationConfig, bool) -> bool
+    """Returns whether distributed tracing is enabled for this integration config"""
+    if "distributed_tracing_enabled" in int_config and int_config.distributed_tracing_enabled is not None:
+        return int_config.distributed_tracing_enabled
+    elif "distributed_tracing" in int_config and int_config.distributed_tracing is not None:
+        return int_config.distributed_tracing
+    return default
```

Comment:
```suggestion
    return int_config.get("distributed_tracing_enabled", int_config.get("distributed_tracing", False))
```
Can probably be simplified to this (and remove `default` as an argument)?

Reply:
now I cannot remember why I did it this way... 🤔

Reply:
ah, this is why... good thing I had these tests to remind myself

https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/10227/workflows/9d5a3317-0898-4ce0-ba19-9d75cf6d0c81/jobs/788436

if the value is `None` we want to ignore it and move on to the next possible one.